app/models/podcast_appearance.rb

```diff
@@ -0,0 +1,11 @@
+class PodcastAppearance < ApplicationRecord
```

Comment:
should this be renamed to `PodcastEpisodeAppearance`? After all it's the join of users and podcast episodes, not podcasts